server/test_integration/lineIntegration.test.ts

```diff
@@ -0,0 +1,149 @@
+import request from 'supertest';
+import { describe, test, beforeAll } from '@jest/globals';
+import { app } from '../index';
+import db from '../src/db/db';
+
+
+
+
+
+
+describe('Line API Integration Tests', () => {
+  beforeAll(async () => {
```

Comment:
i think that the creation of DB is ok with the beforeAll while the insert and delete must be in a beforeEach, so the tests have no conflicts each other